test/unit/raft_test_utils.h

```diff
@@ -96,6 +96,8 @@ struct raft_node {
 	 */
 	bool is_work_blocked;
 
+	/** A bitmap of nodes which see the leader. */
+	vclock_map_t witness_map;
```

Comment:
There is an empty line above to separate regular fields from config options. The witness map is not a config option, so lets move it above that empty line. Or drop the member. It is not used anywhere except for assignment.

Reply:
Removed the `witness_map` field from `raft_node` altogether.